.github/workflows/greetings.yml

```diff
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Thank you for raising an issue.'
+        pr-message: 'Thank you for submitting a Pull Request'
```

Comment:
Out of curiosity. Where do these messages appear? 😊 Can we also do something like this:
```suggestion
        issue-message: 'Thank you for raising an issue. Please be detailed and clear along with an acceptance criteria.'
        pr-message: 'Thank you for submitting a Pull Request. Make sure you have followed all the contribution guidelines.'
```

Reply:
@praveenscience  See this:

![image](https://user-images.githubusercontent.com/56103228/77636493-3723ae80-6f7a-11ea-844f-65c7d91a6605.png)

Reply:
Also, we can surely do that. i'll edit it

Reply:
@mySTRiouscoder Looks good. Can we also have @ mentions? 🤩